app/components/collections/form_component.html.erb

```diff
@@ -75,5 +75,37 @@
     </div>
   </section>
 
+  <section>
+    <header>Review Workflow</header>
+
+    <div class="mb-3 row">
+      <%= form.label :review_enabled, class: 'col-sm-9 col-form-label' do %>
+        A reviewer checks a deposit for clarity, accuracy, and completeness before approving the publication. Only one Reviewer
+        is required to review each deposit. A Reviewer can edit a deposit or return it to the Depositor with a request for changes.
+        The Depositor will receive a notification when their deposit is returned or approved.
+      <% end %>
+      <div class="col-sm-3">
```

Comment:
I'm not saying you need to change this PR, but this is how you can render a switch in Bootstrap: https://getbootstrap.com/docs/4.2/components/forms/#switches

Reply:
Let's make this a new ticket.

Reply:
@jcoyne OK. You want it or shall I?

Reply:
Done: https://github.com/sul-dlss/happy-heron/issues/399

Reply:
thanks!  working on it in a follow on PR now...

Reply:
looks like i should refresh the page proactively before replying 😄